rhel7/profiles/cis.profile

```diff
@@ -625,6 +625,10 @@ selections:
     ### 5.2.15 Ensure SSH warning banner is configured (Scored)
     - sshd_enable_warning_banner
 
+    ### 5.2.18 Ensure SSH MaxStartups is configured (Scored)
+    - sshd_set_maxstartups
+    - var_sshd_set_maxstartups=10:30:60
```

Comment:
The build process is struggling particularly where I have configured this variable. I tried quoting the strings but it appears to fail still.

Strictly speaking, we don't _need_ to specify the variable in the profile itself, as this is already the value that will be set by the rule, but I wanted to configure it in the actual profile so that the CIS-recommended value is always set even if the rule defaults change elsewhere in the project in future.

Reply:
What happens is that `10:30:60` is not a selector for the variable, but the value of the variable. (Also, `default` is not a valid selector).
To explicitly list the variable selection in the profile you to need add a selector with the same value.
Check the example below, `default` has the same value as the `no` selector.
https://github.com/ComplianceAsCode/content/blob/master/linux_os/guide/services/ssh/ssh_server/var_sshd_disable_compression.var

Reply:
Ah, I had assumed that the variable value entered into the `Profile` was a "freeform" field that the user could enter anything into, but now I've had a look at a few other `vars` files I understand how this works now. Thanks!

Reply:
@yuumasato Interestingly though I seem to have still broken the CI job even though I tried quoting the strings in the `Profile` when explicitly setting the variable:
```
KeyError: 'var_sshd_set_maxstartups=10'
```
It seems to hit the `:` character and ignore the remainder of the line, even single-quoted.

Is there something I can do to escape the colon character for the build process, or should I just remove these lines from the `Profile` files? We don't technically need them in there, but it'll help the CIS configuration not drift if the rule gets updated in future for other standards.

Reply:
So the issue is that `:` is the delimiter used to designate control selections.
But this is a variable selection and we should be able to not mistake if for a control.
I filed #7073 to address this issue.

Reply:
Great. Thanks for doing that! We can keep this PR open until #7073 is merged and the CI jobs are able to run properly.